README.md

```diff
@@ -69,6 +69,8 @@ EXAMPLE
   [percy] Finalized parallel build.
 ```
 
+_See code: [dist/commands/finalize.ts](https://github.com/percy/percy-agent/blob/v0.0.50/dist/commands/finalize.ts)_
```

Comment:
Not sure why this was missing from master. But regenerating the docs has added it.